hnn_core/optimization.py

```diff
@@ -528,6 +543,10 @@ def optimize_evoked(net, tstop, n_trials, target_dpl, initial_dpl, maxiter=50,
     print("Initial RMSE: %.2e" % best_rmse)
 
     opt_params = dict()
+
+    if return_rmse is True:
+        opt_params['iter_avg_rmse'] = []
```

Comment:
```suggestion
        opt_params['iter_avg_rmse'] = list()
```
Best practice is to instantiate objects using their official constructors whenever possible.